Dikke Ploaten/Controller/SearchViewController.swift

```diff
@@ -103,19 +102,18 @@ extension SearchViewController {
 		
 		if Database.shared.cache.user.plates.first(where: {$0.albumID == album.id}) != nil {
 			// Already in plates, show remove item
-			let remove = UITableViewRowAction(style: .default, title: "\u{232B}\n Remove", handler: {
-				(action, index) in
+			let remove = UITableViewRowAction(style: .default, title: "\u{232B}\n Remove", handler: { _, _ in
 				// Write instance to database
 				Database.shared.deleteCollectionAlbum(albumId: album.id) { err in
 					if let err = err {
 						print(err)
-						let alertController = UIAlertController(title: "Whoops", message: err.localizedDescription, preferredStyle: .alert)
+						let alertController = UIAlertController(title: "whoops".localized(), message: err.localizedDescription, preferredStyle: .alert)
 						alertController.addAction(UIAlertAction(title: "OK", style: .cancel, handler: nil))
 						self.present(alertController, animated: true, completion: nil)
 					}
 					
 					//Show toast alert
-					self.showToast(controller: self, message: "'\(album.title)' by \(album.artist) was removed from your collection", seconds: 2)
+					self.showToast(controller: self, message: "notification_added".localized(), seconds: 2)
```

Comment:
Hier werk je best met "%album" en "%artist" of zo in je localized string, die je dan hier vervangt door het album en de artist.